tests/monitors/collectd_mysql/mysql_test.py

```diff
@@ -0,0 +1,98 @@
+import time
+from functools import partial as p
+
+import mysql.connector
+
+import pytest
+from tests.helpers.agent import Agent
+from tests.helpers.assertions import has_datapoint_with_dim, has_datapoint_with_metric_name, tcp_socket_open
+from tests.helpers.metadata import Metadata
+from tests.helpers.util import container_ip, ensure_always, run_container, wait_for
+from tests.helpers.verify import (
+    verify,
+    verify_expected_is_subset,
+    run_agent_verify_included_metrics,
+    run_agent_verify_all_metrics,
+)
+
+pytestmark = [pytest.mark.docker_container_stats, pytest.mark.monitor_without_endpoints]
```

Comment:
Sorry I didn't notice this before but looks like the docker mark got copied over here. Otherwise looks good.

Reply:
Oops! my bad. Just fixed it. Thanks for catching!